services/navigation/builtin/builtin.go

```diff
@@ -86,6 +87,15 @@ func (conf *Config) Validate(path string) ([]string, error) {
 		conf.DegPerSec = degPerSecDefault
 	}
 
+	// ensure obstacles have no translation
+	for _, obs := range conf.Obstacles {
```

Comment:
[opt] How would folks feel about an `extra` flag that would allow this? Error by default, but if the user passes `extra["allowObsTranslation"] = true` then it doesn't error.

Reply:
Yeah Peter I think doing what Nick has done is the correct thing for now.  If it happens that we want to expose this down the line then it will be really easy to, but I do agree with the product decision that allowing users to mix these is a little unintuitive